### DISCUSSION TITLE
Explicit content advertising on github.io

### DISCUSSION THREAD

**Discussion by ilyassakhanov** [community/community#78165] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi there, I stumbled upon a page that advertised explicit  content site using github.io . Is it acceptable and under TOS? If not, where do I report it?
[removed for member safety]

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @ilyassakhanov ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

I passed the site along to our team to review and removed the site from your post as to ensure member safety in case anyone does click on the link. In the future these links may be of us: 

- [GitHub Pages Prohibited Uses](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages#prohibited-uses)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
